stetho/src/main/java/com/facebook/stetho/dumpapp/ArgsHelper.java

```diff
@@ -23,4 +25,12 @@ public static String nextArg(Iterator<String> argsIter, String errorIfMissing)
     }
     return argsIter.next();
   }
+
+  public static String[] argsIterToArray(Iterator<String> argsIter) {
```

Comment:
Prefer a name consistent with the other methods in this class such as `toArray`.  All `ArgsHelper` methods assume you have an `Iterator<String>`.